server/middleware.ts

```diff
@@ -54,6 +54,27 @@ export async function middleware(request: NextRequest) {
     },
   );
 
+  if (process.env.NODE_ENV === "development" && process.env.BYPASS_AUTH === "true") {
+    const dummyEmail: string = process.env.DUMMY_EMAIL!;
+    const dummyPassword:string = process.env.DUMMY_PASSWORD!;
+    const { data, error: signInError } = await supabase.auth.signInWithPassword({
+      email: dummyEmail,
+      password: dummyPassword,
+    })
+
+    if (signInError) {
+      const { error: signUpError } = await supabase.auth.signUp({
+        email: dummyEmail,
+        password: dummyPassword
+      });
```

Comment:
@akoshelev instead of putting this in `seed.sql`, it's created on the fly the first time, but only in the development mode, and with `BYPASS_AUTH` turned on.

Reply:
that's much better!